src/test/java/common/bankarskiSistem/unit/service/UserServiceUnitTests.java

```diff
@@ -1,13 +1,61 @@
 package common.bankarskiSistem.unit.service;
 
+import common.bankarskiSistem.exceptions.EntityNotFoundException;
+import common.bankarskiSistem.model.ExchangeRates;
+import common.bankarskiSistem.model.User;
+import common.bankarskiSistem.repository.ExchangeRatesRepository;
+import common.bankarskiSistem.repository.UserRepository;
+import common.bankarskiSistem.service.BankService;
+import common.bankarskiSistem.service.UserService;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.NullSource;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
 
 @SpringBootTest
 class UserServiceUnitTests {
 
-	@Test
-	void contextLoads() {
+	@MockBean
+	private UserRepository userRepository;
+
+
+	@Autowired
+	private UserService userService;
+
+	@ParameterizedTest
+	@MethodSource({"common.bankarskiSistem.unit.service.parametrized.UserParameters#generateUpdate"})
+	void canUpdateUser(User userOld, User userNew){
+
+		when(userRepository.getReferenceById(userOld.getPersonalId())).thenReturn(userOld);
+		when(userRepository.save(any(User.class))).thenReturn(userNew);
+
+		User result= null;
```

Comment:
Ne bi trebala ovde da bude obrada izuzetaka jer do njega nece nikad doci, to znamo, pa se moze samo delegirati da nas test baca izuzetak (jer ga nikad nece ni baciti). Test ce biti kraci i lakse citljiv